package.json

```diff
@@ -14,6 +14,9 @@
     "cross-tab"
   ],
   "scripts": {
+    "demo": "vite serve ./demo --port 3000",
+    "demo:build": "vite build ./demo",
```

Comment:
Why we need to build `demo` if we need only tro run it for manual tests?

Reply:
Because otherwise you would not be able to catch bfcache behaviour. I don't know why but in "dev" mode it does not hit it at all. Only in build mode.
I can investigate further into this if needed.

Reply:
What script we need right now to test bfcache? Let’s make the single `start` script (for instance, it will compile files and run server)

Reply:
Done